### ISSUE TITLE
Allow existing components to be modified using lwc-builder

### ISSUE THREAD

**Issue by rahulgawale** [developerforce/lwc-builder#108]
If there is an existing component we can't edit that directly with builder extension. Need an option to open an existing component and preload the existing configuration, so we can modify that and save it back to the component meta.xml file. I am planning to work on it. Let me know your thoughts.

**Comment by dmarino324** [comment]
Yes this would a much appreciated function. As it is now this is only useful for initial creation. But still VERY useful.
